content/docs/dvclive/integrations/mmcv.md

```diff
@@ -0,0 +1,64 @@
+# MMCV
+
+The DVCLive - MMCV integration allows you to easily add experiment tracking
+capabilities to your OpenMMlab projects.
+
+The integration is
+[maintained in the MMCV repository](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/logger/dvclive.py).
```

Comment:
Are we managing this? Do we want tests for it?

Reply:
I was thinking about how to handle this integrations and I think that it makes sense to have them maintained as part of the `dvclive` repository:

https://github.com/iterative/dvclive/pull/125